lms/templates/courseware/progress.html

```diff
@@ -57,8 +57,9 @@ <h1 class="progress-certificates-title">${_("Course Progress for Student '{usern
 
       <section class="progress-info-mouseover">
         <p>
-            Note: Submitted assignments will appear as bars in the Progress chart.
-            Mouse over a bar to see more information about the relevant assignment.
+            ${_('''Note: Submitted assignments will appear as bars in the Progress chart.
```

Comment:
I think some standards guidelines advise double-quotes for Python docstring syntax?